app/views/users/two_factor_authentication/email/show_resend.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">Resend two-factor authentication email</h1>
+  <p class="govuk-body">
+      Emails sometimes take a few minutes to arrive. If you do not receive the email, we can send
+      you a new one.
```

Comment:
not sure what this newline is for

Reply:
Because it's more than 100 chars, though I'm not sure if we have a standard that defines this.

[Ruby styleguide](https://rubystyle.guide/#max-line-length) suggests 80 chars, but argues that teams could agree on a higher count. Do you know what the consensus is in the community?

Reply:
I don't think I've ever seen it enforced in ERB files, which are usually more HTML than Ruby and usually much more verbose. Also:

```
      Metrics/LineLength:
        Enabled: false
```

in our `.erb-lint.yml`, so I'd vote for a single line as I've had my share of time lost because of whitespace interpretation in templating engines :smile: